apollo-rx2support/src/main/java/com/apollographql/apollo/rx2/Rx2Apollo.java

```diff
@@ -34,14 +34,14 @@ private Rx2Apollo() {
   }
 
   /**
-   * Converts an {@link ApolloWatcher} to an asynchronous Observable.
+   * Converts an {@link ApolloQueryWatcher} to an asynchronous Observable.
    *
    * @param watcher the ApolloWatcher to convert.
```

Comment:
We need to change it to:

 `@param watcher the ApolloQueryWatcher to convert.`